src/renderer/src/components/LeaveProjectModal/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Dialog } from '@mui/material'
+import { useState } from 'react'
+import { LeaveProjectConfirmation } from './LeaveProjectConfirmation'
+import { DeleteData } from './DeleteData'
+import { SuccessfulLeave } from './SuccessfulLeave'
+
+type LeaveProjectModalProps = {
+  isOpen: boolean
+  closeModal: () => void
+}
+
+export const LeaveProjectModal = ({ isOpen, closeModal }: LeaveProjectModalProps) => {
+  function handleCloseDialog(event?: Record<string, never>, reason?: 'escapeKeyDown' | 'backdropClick') {
+    if (reason === 'backdropClick' || reason === 'escapeKeyDown') return
+    closeModal()
+  }
+
+  const [modalContent, setModalContent] = useState<'confirmation' | 'deleteData' | 'successfulLeave'>(
+    'confirmation',
+  )
+
+  return (
+    <Dialog open={isOpen} onClose={handleCloseDialog} maxWidth="xl" fullWidth>
```

Comment:
the dimensions specified for each content child here seems to cause issues i.e. `<Column sx={{ minHeight: '85vh', padding: 4, pt: '10vh' }} ...`

<img width="1791" alt="image" src="https://user-images.githubusercontent.com/18542095/234618007-a230448b-a4d3-42a6-a09d-28e85a3af289.png">

would suggest updating this dialog here to use the `fullScreen` prop and then apply a margin on it. that way you don't have to specify dimensions for the content child's container. example at

https://github.com/digidem/mapeo-desktop-shell/blob/005aa150f3da1589c03a82ce0487be08c6188967/src/renderer/src/views/Home/Settings/ProjectConfig/InviteDeviceModal.tsx#L42-L43

Reply:
https://github.com/digidem/mapeo-desktop-shell/pull/58 should be of interest if we're interested in having some consistency 😄

Reply:
https://github.com/digidem/mapeo-desktop-shell/pull/56/commits/f5d14d3ac957f1e7c82fa841e3608d8a288a3405

Reply:
still seeing some issues with the footer area:

<img width="1791" alt="image" src="https://user-images.githubusercontent.com/18542095/234660212-f93e4e8f-ff71-4e4a-b015-b8cba2e372d3.png">

would suggest removing the `sx` prop from the `Column` in `LeaveProjectConfirmation` and applying the appropriate padding for the sections within that component.

Reply:
Sorry, Im not sure what the problem is that we are trying to fix with the footer